### DISCUSSION TITLE
Whta is the checksum font of Custom Frame in Data Format

### DISCUSSION THREAD

**Discussion by muye998** [hyOzd/serialplot#35] [Q&A] [answered]
Hi,I want to caculate the checksum to make the value getting more procise ,so I need you help ,thank you for your request!

**Comment by hyOzd** [comment, marked as answer]
Checksum is the `least significant (right most) byte of sum of all bytes in the payload not including start bytes or length bytes`. 

When sending your samples you should add all bytes that are being sent and send the resulting sum at the end. Checksum at the moment is only 1 byte, so sending only the least significant byte is enough.

**Reply by muye998** [reply]
Thank you for your reply! It was the length of 1 byte that confused me. Thank you!
